### PR TITLE
New test for the bug reported #17913

### PR DESCRIPTION
**Pull request by davefern**
Wrote a simple test in a try block for issue #17913 and put some more details in comments for others to work on. Will keep trying to fix but wanted the test to be there for others who may be looking at working on it and for perpetuity.

### CODE REVIEW

File: sympy/parsing/tests/test_ast_parser.py [hunk 1]
```diff
@@ -16,3 +16,28 @@ def test_parse_expr():
...
+                , keywords=[]
+            )#Body Call
+        )#Expression
+        
+        But when we pass it into compile we get a type error claiming this object is a AST arg object.
+        
+        expected some sort of expr, but got <_ast.arg object at 0x10075da00>
+        
+        May be an issue with fix_missing_locations?"""
+        print("Lambda AST Parsing Error)
```
Comment (czgdp1807) [inline sympy/parsing/tests/test_ast_parser.py]: Please remove `print` statements from the code.

**Comment by oscarbenjamin** [comment]
If this is a test that should pass when fixed it should use the XFAIL decorator.

**Comment by czgdp1807** [comment]
@davefern Would you still work on this PR? Thanks for your contributions.

**Comment by czgdp1807** [comment]
Closing this as of now. However, if needed please feel free to re-open this PR in future. Thanks all for the contributions.

**Pull request closed without merging**